src/component/home.js

```diff
@@ -25,28 +25,35 @@ export default function Home() {
             return (
               <li className="card-vertical" key={video._id}>
                 <div className="card__primary-action card__primary-action-column card__primary-action-vertical">
-                  <span className="material-icons-outlined badge-up-right-corner card-badge card-badge-vertical">
-                    {likeState.likeVideos &&
-                    likeState.likeVideos.some(
-                      (item) => video._id === item._id
-                    ) ? (
-                      <AiTwotoneLike
-                        color="red"
-                        onClick={() =>
-                          likeDispatch({
-                            type: "REMOVE_FROM_LIKE",
-                            payload: video
-                          })
-                        }
-                      />
-                    ) : (
-                      <AiTwotoneLike
-                        onClick={() =>
-                          likeDispatch({ type: "ADD_TO_LIKE", payload: video })
-                        }
-                      />
-                    )}
-                  </span>
+                  {loginStatus ? (
+                    <span className="material-icons-outlined badge-up-right-corner card-badge card-badge-vertical">
+                      {likeState.likeVideos &&
+                      likeState.likeVideos.some(
+                        (item) => video._id === item._id
+                      ) ? (
+                        <AiTwotoneLike
+                          color="red"
+                          onClick={() =>
+                            likeDispatch({
+                              type: "REMOVE_FROM_LIKE",
+                              payload: video
+                            })
+                          }
+                        />
+                      ) : (
+                        <AiTwotoneLike
+                          onClick={() =>
+                            likeDispatch({
+                              type: "ADD_TO_LIKE",
+                              payload: video
+                            })
+                          }
+                        />
+                      )}
+                    </span>
+                  ) : (
+                    ""
```

Comment:
Just a suggestion you could have add a empty fragment instead of empty string

Reply:
thanks i will do that